.github/workflows/coverage-pr.yml

```diff
@@ -0,0 +1,148 @@
+name: Coverage (PR)
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+permissions:
+  contents: read
+  pull-requests: write
+  actions: read
+
+jobs:
+  cov-head:
+    runs-on: ubuntu-latest
+    container:
+      image: rust:1.88
```

Comment:
Can we avoid hardcoding rust version here and below? And rely on https://github.com/NethermindEth/rust-template/blob/main/rust-toolchain.toml instead?